packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -201,6 +202,13 @@ class ExpansionTile extends StatefulWidget {
   /// which means that the expansion arrow icon will appear on the tile's trailing edge.
   final ListTileControlAffinity? controlAffinity;
 
+  /// Specifies if the list tile is expanded (true) or collapsed (false, the default).
+  ///
+  /// If both [isExpanded] and [initiallyExpanded] will be provided,
+  /// the value of [isExpanded] will be considered.
+  /// Typically used for programmatically control the expansion of the tile.
+  final bool? isExpanded;
```

Comment:
The problem with this flag is that it can get out of sync. Let's say this is set to true. Now the user collapses the tile by tapping on it. The flag will still be true even though in the internal state it has switched to false. If you now want to programmatically expand the tap again, you're out of luck because the flag is already set to true and you can't re-set it to true.

Reply:
I fixed it in the last commit. Should I reopen the PR?

Reply:
> @goderbauer I fixed it in the last commit. Should I reopen the PR?

Please PR it again. This is something a lot of people would love to see!